tests/e2e/framework/framework_test.go

```diff
@@ -53,7 +53,7 @@ func newCommonConfig(testID string) (*CommonConfig, error) {
 	if err != nil {
 		return nil, err
 	}
-	k, err := newKubeInfo(t.TempDir, t.RunID)
+	k, err := newKubeInfo(t.TempDir, t.RunID, "")
```

Comment:
Can we have another function called `newKubeInfoWithBaseVersion(tmpDir, runID, baseVersion string)` and keep the original `newKubeInfo(tmpDir, runID string)` which calls `newKubeInfoWithBaseVersion()` with the empty string as baseVersion?

Reply:
Done

Reply:
Reverted as changing that caused a lint error (newKubeInfo with empty string is not used by any code)